pre-commit.nix

```diff
@@ -0,0 +1,9 @@
+{pkgs, ...}: let
```

Comment:
Is this just for installing the hook? Hopefully, we still have the pre-commit in yaml for those contributing who want to use nix (and we want to check against their work).

Reply:
Yes it's still in the YAML file